### PR TITLE
feat(Spanner): batch DDL statements in CreateDatabase sample

### LINKED ISSUE #1251: [Spanner] CreateDatabase sample should batch all DDL statements
The current `CreateDatabase` sample for Cloud Spanner executes three separate DDL statements to create a database with 2 tables. It is [generally advisable to batch DDL statements](https://cloud.google.com/spanner/docs/schema-updates#best-practices) together, both to speed up the execution of the DDL statements and to reduce the number of DDL statements that is sent to Spanner.

The [create database samples](https://cloud.google.com/spanner/docs/samples/spanner-create-database) for the other client libraries all use batched DDL statements.

### PR DESCRIPTION
**Pull request by olavloite**
Batch DDL statements to reduce execution time and reduce the number of DDL requests to Spanner.

Fixes #1251

### CODE REVIEW

**Comment by amanda-tarafa** [comment]
nit: (I've changed the title to feat(Spanner), in the samples repo, adding or improving a sample is a feature and not a documentation improvement, or that's how we've been interpretting it, although we are not very strict about it, so happy to revert back if you disagree)

File: spanner/api/Spanner.Samples/CreateDatabaseAsync.cs [hunk 1]
```diff
@@ -23,41 +23,27 @@ public async Task CreateDatabaseAsync(string projectId, string instanceId, strin
...
-        connectionString += $"/databases/{databaseId}";
-        using (var connection = new SpannerConnection(connectionString))
-        {
-            // Define create table statement for table #1.
-            string createTableStatement =
-            @"CREATE TABLE Singers (
+        using var connection = new SpannerConnection(connectionString);
+        var createStatement = $"CREATE DATABASE `{databaseId}`";
+        // Define create table statement for table #1.
+        var createTable1 =
```
Comment (amanda-tarafa) [inline spanner/api/Spanner.Samples/CreateDatabaseAsync.cs:29]:
```suggestion
        var createSingersTable =
```

File: spanner/api/Spanner.Samples/CreateDatabaseAsync.cs [hunk 2]
```diff
@@ -23,41 +23,27 @@ public async Task CreateDatabaseAsync(string projectId, string instanceId, strin
...
-
-        // Update connection string with Database ID for table creation.
-        connectionString += $"/databases/{databaseId}";
-        using (var connection = new SpannerConnection(connectionString))
-        {
-            // Define create table statement for table #1.
-            string createTableStatement =
-            @"CREATE TABLE Singers (
+        using var connection = new SpannerConnection(connectionString);
+        var createStatement = $"CREATE DATABASE `{databaseId}`";
```
Comment (amanda-tarafa) [inline spanner/api/Spanner.Samples/CreateDatabaseAsync.cs:27]:
```suggestion
        var createDatabase = $"CREATE DATABASE `{databaseId}`";
```

File: spanner/api/Spanner.Samples/CreateDatabaseAsync.cs [hunk 3]
```diff
@@ -23,41 +23,27 @@ public async Task CreateDatabaseAsync(string projectId, string instanceId, strin
...
                  ) PRIMARY KEY (SingerId)";
-
-            using var createTableCommand = connection.CreateDdlCommand(createTableStatement);
-            await createTableCommand.ExecuteNonQueryAsync();
-
-            // Define create table statement for table #2.
-            createTableStatement =
-            @"CREATE TABLE Albums (
+        // Define create table statement for table #2.
+        var createTable2 =
```
Comment (amanda-tarafa) [inline spanner/api/Spanner.Samples/CreateDatabaseAsync.cs:37]:
```suggestion
        var createAlbumsTable =
```

**Review by amanda-tarafa — APPROVED** [review]
LGTM. Just var name nits.

**Review by skuruppu — APPROVED** [review]

**Comment by olavloite** [comment]
@amanda-tarafa Would you mind taking a quick look at the above build failure? As far as I can see, they are not related to this change. I don't have permission to add labels in this repository, so I can't force a build (without making a dummy change).

**Comment by amanda-tarafa** [comment]
Yes, totally unrelated. Expect that to happen on this repo from time time. We are slowly working on fixing the flakiness.
Let me know if this is ready for merge and I can force merge.

**Comment by olavloite** [comment]
As far as I'm concerned, this is ready to be merged.

**Pull request merged**